src/index.ts

```diff
@@ -1,18 +1,28 @@
 import Bluebird = require('bluebird');
 import * as dotenv from 'dotenv';
+import schedule = require('node-schedule');
 dotenv.config();
 import * as Koa from 'koa';
 import * as bodyParser from 'koa-bodyparser';
 import * as jwt from 'koa-jwt';
 import * as logger from 'koa-logger';
 import * as passport from 'koa-passport';
 import * as mongoose from 'mongoose';
+import { db, port, triggerSchedule } from './config/config';
 import config from './config/test';
 import routes from './routes';
+import { getInfo } from './utils/parserUtils';
 
 const server = new Koa();
+// run parse api whe server started
+getInfo();
 
-mongoose.connect(process.env.DB, { useMongoClient: true });
+// run sheduled parser task
+const parse = schedule.scheduleJob(triggerSchedule, () => {
+  getInfo();
```

Comment:
It is not called now, it will be called after 3  days.When we start or restart (after crash server) server, first call updates db or fill it if db is empty, second call will be after 3 day.

Reply:
name of method fixed =)